src/createServer.js

```diff
@@ -1,11 +1,215 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  let expenses = [];
+  let users = [];
+
+  app.use(express.json());
```

Comment:
if u add here '/users' or'/expenses' - so u don't have to write the whole path in each request